cli-output/src/cli_output.rs

```diff
@@ -1352,6 +1352,71 @@ pub fn parse_sign_only_reply_string(reply: &str) -> SignOnly {
     }
 }
 
+#[derive(Serialize, Deserialize)]
```

Comment:
Could these things live in `cli/src/feature.rs`?    I really like how `cli/src/feature.rs` exposes a minimal surface area to the rest of the cli code

Reply:
Yes, I agree! I had considered busting up cli_output as a follow-up sweeping change, but I'll go ahead and move these now.

Reply:
Updated

Reply:
> I had considered busting up cli_output

+1 Plenty of stuff in here that's CLI-specific and will never be useful elsewhere